target/linux/realtek/dts-5.10/rtl839x.dtsi

```diff
@@ -170,6 +170,15 @@
 			#size-cells = <0>;
 		};
 
+		timer0: timer@3100 {
+			compatible = "realtek,rtl8390-timer";
+			reg = <0x3100 0x10>, <0x3110 0x10>, <0x3120 0x20>;
```

Comment:
Hmm, I thought you had 2 address blocks per timer; so why a) is there a 3rd (or 2times 1) and b) would that not be timer1?

Same goes for the interupt below, shouldn't that be the interrupt for the second timer?